tests/integration/network.selectors.ts

```diff
@@ -0,0 +1,10 @@
+export const NetworkSelectors = {
+  NetworkName: 'network-name',
+  NetworkAddress: 'network-address',
+  NetworkKey: 'network-key',
+  BtnAddNetwork: 'btn-add-network',
+  ErrorText: 'error-text',
+  APINetwork: process.env.APIEnvVariable || 'https://stacks-node-api-inactive.stacks.co/',
```

Comment:
These `XXXSelectors` objects are really for CSS Selectors to target HTML elements: buttons, dropdowns etc, used within `data-testid` attributes. 

We shouldn't be adding non-selector values, such as network, in here.

Reply:
I can add one long test that does this but then you wouldn't be able to see what asset has the issue easily when the test fails. The reason for multiple scripts is because I wanted to highlight when MIA coin breaks or when USDA breaks easily.

Reply:
> The reason for multiple scripts is because I wanted to highlight when MIA coin breaks or when USDA breaks easily. 

I get it. Though there's no reason we can't make the logs clear that only one has broken. Given how uncommon this scenario should be, I think that's better than having 4x the same duplicate code to maintain.